packages/utilities/psammead-storybook-helpers/src/input-provider.js

```diff
@@ -0,0 +1,18 @@
+import { boolean, text, select } from '@storybook/addon-knobs';
+import { LANGUAGE_VARIANTS } from './text-variants';
+
+/* eslint-disable import/prefer-default-export */
+export const inputProvider = (names, componentFunction) => () => {
+  const inputs = names.map(name =>
+    boolean(`Show presets for ${name}`, false, name)
+      ? select(
+          `Content for ${name}`,
+          LANGUAGE_VARIANTS,
+          LANGUAGE_VARIANTS.News,
```

Comment:
You should be able to remove this line
```suggestion
```

Reply:
Hey Sareh. This value is the default selected on the `select` dropdown. While it will not be removed, we need to update it to `LANGUAGE_VARIANTS.English` now. Going to do just that.

Reply:
```suggestion
          LANGUAGE_VARIANTS.English,
```